goal_src/jak2/engine/process-drawable/process-focusable.gc

```diff
@@ -76,7 +76,10 @@
                       (-> (the-as collide-shape-moving gp-0) gspot-pos)
                       )
                      ((and (or (= arg0 2) (= arg0 3)) (type? gp-0 collide-shape))
-                      (-> gp-0 root-prim prim-core)
+                      ;; PC PORT NOTE : added lod-set check so we don't use invalid bones
+                      (if (not (logtest? (-> obj draw status) (draw-control-status lod-set)))
+                          (-> gp-0 trans)
+                          (-> gp-0 root-prim prim-core))
```

Comment:
I'm still not totally convinced this is right... I think there could be another bug somewhere else because I would never expect the `(-> gp-0 root-prim prim-core trans)` to be NaN. I guess let's merge this for now and I will look into it later: https://github.com/open-goal/jak-project/issues/2581 if we find a better fix maybe we can remove this change.